src/main/java/com/sforce/ws/bind/TypeMapper.java

```diff
@@ -66,6 +67,8 @@ public class TypeMapper {
     private static final HashSet<String> keywords = getKeyWords();
     private static HashMap<String, Class<?>> primitiveClassCache = getPrimitiveClassCache();
 
+    private static final DateTimeFormatter DTF = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSX");
```

Comment:
Is the final X correct? If the OffsetDateTime value always has zero offset, then this should work. However, if non-zero offsets are possible, then we probably want XXX to match [Salesforce](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/intro_valid_date_formats.htm). Note that [OffsetDateTime.toString()](https://docs.oracle.com/javase/8/docs/api/java/time/OffsetDateTime.html#toString--) uses XXXXXX.

Reply:
Let me change it and give it a test